backendAPI/views.py

```diff
@@ -6,7 +6,7 @@
 
 def check_login(request):
     try:
-        if request.GET.get('isOAuth', False):
+        if request.GET.get('isOAuth') == 'false':
             username = request.GET['username']
             password = request.GET['password']
             user = User.objects.get(username=username, password=password)
```

Comment:
password is stored as a hash in django database user table, this needs to be changed. See this https://stackoverflow.com/questions/4822724/check-password-from-a-user-again